### PR TITLE
trezor-agent 0.9.3 (version bump)

### PR DESCRIPTION
**Pull request by cerebralmisfire**
Some of the key libraries have been updated to work with the new hardware.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/trezor-agent.rb [hunk 1]
```diff
@@ -5,12 +5,14 @@ class TrezorAgent < Formula
...
   url "https://files.pythonhosted.org/packages/54/89/6de88988f9efa712cb7131cf3123a2a043c7956ee66e725b14a1ea4c27ec/trezor_agent-0.9.2.tar.gz"
   sha256 "d7756759ab9bb6700007423e58cb20f26effd0271c26f7a43c0a92c0fa487ede"
+  revision 1
 
   bottle do
     cellar :any_skip_relocation
-    sha256 "ff4b2a030026867a0895f99d63a5a1010cf8b1699b093c91e99b92e1d2cd8744" => :high_sierra
-    sha256 "a9c354d429d49c1fabd255f18548f03967f086f136de8ab8a7bef0eb4e7efbb8" => :sierra
-    sha256 "2da90be8d971b100a37ad9b6e64d64e3ced33a86c79a940e2e912f86d1bc4682" => :el_capitan
+    sha256 "0eab1af044dca8d5e568bf6252c06a8daef038a8d8f95bac4832b7075c00aa67" => :mojave
```
Comment (DomT4) [inline Formula/trezor-agent.rb]: Please rebase this against master so you're not accidentally touching the bottle block, thanks.

**Review by DomT4 — CHANGES_REQUESTED** [review]

**Comment by DomT4** [comment]
We don't usually update resources out-of-band; what's the justification here?

**Comment by menteb** [comment]
The firmware on the Trezor has been updated and new devices were launched. Hence, the libraries used to communicate with the hardware had to be updated.

**Review by cerebralmisfire — COMMENTED** [review]
Cheers

**Comment by DomT4** [comment]
Okay, that seems reasonable enough justification to go ahead here. You'll need to fix that touching of the `bottle do` block though, otherwise git will panic when someone tries to fold this in.

File: Formula/trezor-agent.rb (hunk 1 above)
Comment (menteb) [reply Formula/trezor-agent.rb]: I'm afraid I don't understand. Not a programmer really, just trying to help out a project...

File: Formula/trezor-agent.rb [hunk 2]
```diff
@@ -5,12 +5,14 @@ class TrezorAgent < Formula
   homepage "https://github.com/romanz/trezor-agent"
   url "https://files.pythonhosted.org/packages/54/89/6de88988f9efa712cb7131cf3123a2a043c7956ee66e725b14a1ea4c27ec/trezor_agent-0.9.2.tar.gz"
```
Comment (javian) [inline Formula/trezor-agent.rb]: The latest version is v0.11.3 according to the github page , you should update that as well.

Comment (DomT4) [reply Formula/trezor-agent.rb]: 0.9.2 is still the one on Pypi. Not sure what the situation is with that.

Comment (javian) [reply Formula/trezor-agent.rb]: Hmm, the release version on the github page and the ones on pythonhosted doesn't seem to match up with dates or anything.

Comment (DomT4) [reply Formula/trezor-agent.rb]: Seems possible they use GitHub as a sort of rolling development preview and PyPI for stable releases. Potentially worth asking upstream directly @menteb, unless you know the answer yourself.

Comment (javian) [reply Formula/trezor-agent.rb]: Possible. The 0.9.1 version and 0.9.2 version on pythonhosted lags by about a year though but 0.9.0 is release on the same day.

Comment (menteb) [reply Formula/trezor-agent.rb]: I'll contact the developer

**Review by cerebralmisfire — COMMENTED** [review]
Version now bumped to 0.9.3 after developer pushed it to pythonhosting.

**Review by cerebralmisfire — COMMENTED** [review]
Should be fine?

**Pull request closed without merging**